pkg/machine/qemu/options_windows_amd64.go

```diff
@@ -0,0 +1,18 @@
+package qemu
+
+var (
+	QemuCommand = "qemu-system-x86_64"
+)
+
+func (v *MachineVM) addArchOptions() []string {
+	opts := []string{"-machine", "q35,accel=whpx:tcg", "-cpu", "max,vmx=off,monitor=off"}
```

Comment:
minor nit: Could you add a comment here explaining like you did in the PR why certain flags have to be disabled and maybe a bug ref if you think its pertinent (e.g. https://gitlab.com/qemu-project/qemu/-/issues/628 for vmx=off)

Reply:
can be in a later PR if you prefer, not blocking to me